src/modules/extendedBuilding/assets/enhancedPersonnelAssignment.ts

```diff
@@ -8,132 +9,230 @@ export default async (
     setSetting: Parameters<ModuleMainFunction>[0]['setSetting'],
     $m: $m
 ): Promise<void> => {
-    await LSSM.$stores.api.getBuildings(
+    // extract the vehicle ID out of the URL
+    const vehicleId = parseInt(
+        window.location.pathname.match(/\d+(?=\/zuweisung)/u)?.[0] || '-1'
```

Comment:
Is there any case, where the url could have unicode included? Otherwise the `/u` seems a little bit useless.

Reply:
Makes sense. But except the korean version we should be good to go with extend ASCII.